src/utils_project.js

```diff
@@ -284,8 +291,9 @@ go.utils_project = {
                         'Connection': ['close']
                     }
                 });
+
                 return http
-                    .head(voice_url)
+                    .head(voice_url[0])
```

Comment:
@imsickofmaps what would you suggest here in terms of logging and doing the http head request when there're two voice files in question..?  As you can see, currently I only use the first voice file in the array